examples/lighting-app/efr32/src/AppTask.cpp

```diff
@@ -65,6 +69,12 @@ static bool sIsThreadEnabled         = false;
 static bool sHaveBLEConnections      = false;
 static bool sHaveServiceConnectivity = false;
 
+uint8_t sAppEventQueueBuffer[APP_EVENT_QUEUE_SIZE * sizeof(AppEvent)];
```

Comment:
Can these go in an unnamed namespace so they will have internal linkage?

`sHaveBLEConnections` et al. can also move, and then we won't need the `static` qualifier